### ISSUE TITLE
Be careful with this VPN

### ISSUE THREAD

**Issue by Happ1ness-dev** [dogwifhatdvpndev/dogwifhat-android#2]
Hello, I just want to warn you that this VPN has some weird management who are banning people for absolutely no reason in their Telegram group (@dogwifhatdvpn).

In my case, I just asked questions about locally building the VPN and got my message deleted. When I asked why, they told me it's off-topic and I should ask in another group (@SentinelDVPN). I then proceeded to ask the same question there, and, to my surprise, I found out that my Telegram account had been restricted for spam.

~~My theory is that the admin of the group (@dogwifhatdvpn), whether accidentally or on purpose, did not uncheck the 'report for spam' option when deleting my first message, which resulted in my Telegram account being restricted.~~
No, they most likely didn't. The false account restriction was probably a coincidence, because otherwise I'd not be able to send messages in the group I was reported in. I'm still banned in @dogwifhatdvpn group though.

When I told them that I can't access the group (@SentinelDVPN) they had directed me to, it looked like they initially tried to help me, but then banned me a day later without any prior warnings.

TLDR:
They banned me in their group (@dogwifhatdvpn) for asking innocent questions.
